app/shared/models/git_repo.rb

```diff
@@ -363,30 +363,32 @@ def reset_hard!(use_global_git_mutex: true)
     # This method commits and pushes all changes
     # if `file_to_commit` is `nil`, all files will be added
     # TODO: this method isn't actually tested yet
-    def commit_changes!(commit_message: nil, file_to_commit: nil, repo_auth: self.repo_auth)
+    def commit_changes!(commit_message: nil, push_after_commit: true, file_to_commit: nil, repo_auth: self.repo_auth)
       git_action_with_queue do
         logger.debug("Starting commit_changes! #{self.git_config.git_url} for #{repo_auth.username}")
         raise "file_to_commit not yet implemented" if file_to_commit
         commit_message ||= "Automatic commit by fastlane.ci"
 
-        self.setup_author(full_name: repo_auth.full_name, username: repo_auth.username)
+        setup_author(full_name: repo_auth.full_name, username: repo_auth.username)
```

Comment:
any specific reason for this?

Reply:
what part of `this` are you referring to?

Reply:
Oh sorry, I meant the change of this particular line, being removing the `self` when calling `self.author`

Reply:
Ahhh yeah, I'm starting to get used to doing things the `ruby way` which is not using self to refer to stuff.